docs/modify-odh-deployment.md

```diff
@@ -0,0 +1,19 @@
+# Modify your ODH deployment
```

Comment:
What are we trying to modify here exactly? ODH itself? Or apps that being deployed?

Reply:
We are going to modify the deployment, meaning the components that are deployed.